### DISCUSSION TITLE
mkdocs material not working properly

### DISCUSSION THREAD

**Discussion by aprilahijriyan** [squidfunk/mkdocs-material#2423] [Q&A] [answered]
Hi, I want to ask why my site doesn't really look like it uses a material theme?
my site: https://ngedev.github.io/

And also you can see the source at https://github.com/ngedev/ngedev.github.io

I think the mkdocs configuration is correct as in the documentation.
Thank you in advance.

**Comment by facelessuser** [comment]
Works for me when I run it locally. Must be something you are doing, but you would have to provide more information:

<img width="1220" alt="Screen Shot 2021-03-09 at 9 41 43 PM" src="https://user-images.githubusercontent.com/1055125/110577696-677b4300-8120-11eb-9c26-595a72c9755f.png">

**Reply by aprilahijriyan** [reply]
That's right, the material theme runs normally on my local computer too. But when I deploy to github page. It's not working.

**Reply by aprilahijriyan** [reply]
![Screen Capture_20210310115215](https://user-images.githubusercontent.com/37798612/110578324-1e1a0a80-8197-11eb-870e-f9d029a904d0.png)

**Comment by martinbira** [comment, marked as answer]
Now I've never deployed with GitHub Actions myself, but taking a quick glance at the documentation, but the file you named as `mkdocs.yml` in workflows should maybe be called `ci.yml`. The `mkdocs.yml` file is for Mkdocs, not GitHub Actions.
https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions

**Reply by aprilahijriyan** [reply]
Good understanding. But I've tried to change the workflow name 2 times. still not working.

**Reply by aprilahijriyan** [reply]
See the github action log https://github.com/ngedev/ngedev.github.io/actions

**Reply by squidfunk** [reply]
Also make sure that in the [repository settings](https://github.com/ngedev/ngedev.github.io/settings), the correct branch name is set:

<img width="926" alt="Bildschirmfoto 2021-03-10 um 09 39 49" src="https://user-images.githubusercontent.com/932156/110600790-9c20e600-8184-11eb-9cc0-6bdab5591d95.png">

On a side note, some users reported problems with GitHub Pages in the last few days - could be related.

**Reply by aprilahijriyan** [reply]
Wow! thanks for the help, it's working!
